client/post-editor/editor-sharing/accordion.jsx

```diff
@@ -28,6 +28,8 @@ import { getSiteUserConnections } from 'state/sharing/publicize/selectors';
 import { postTypeSupports } from 'state/post-types/selectors';
 import { fetchConnections as requestConnections } from 'state/sharing/publicize/actions';
 
+require('./style.scss');
```

Comment:
Not convinced that the saving of up to 1 line per jsx file is worth this magic.

Developers understand specifying module dependencies in code. Our problem right now is that the CSS dependencies are specified in a separate tree from JS (component) dependencies. If we solve that, I think having localized requires is fine.

if we want to use CSS Modules at some point, we'd also need the imported object